edk2toollib/utility_functions.py

```diff
@@ -264,7 +266,6 @@ def DetachedSignWithSignTool(SignToolPath, ToSignFilePath, SignatureOutputFile,
     params = params + ' /debug /v "' + ToSignFilePath + '" '
     ret = RunCmd(SignToolPath, params)
     if(ret != 0):
-        logging.error("Signtool failed %d" % ret)
         return ret
```

Comment:
this change will lead to different results because the return code will be printed at info level instead of error.  Probably would prefer to leave this here.

Reply:
Got it.  I'll undo the changes on line 267 and also around line 307 as well, since that's the same thing (I had removed a logging.error print).

Reply:
Got it.  I've gone ahead and reverted this portion of the change, along with a similar change around line 305.